apps/ledger-live-mobile/src/components/RootNavigator/EditTransactionNavigator.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useMemo } from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+import { useTranslation } from "react-i18next";
+import { useTheme } from "styled-components/native";
+
+import { ScreenName } from "../../const";
+import { getStackNavigatorConfig } from "../../navigation/navigatorConfig";
+import StepHeader from "../StepHeader";
+import { MethodSelection } from "../../families/ethereum/EditTransaction/MethodSelection";
```

Comment:
❌  it is forbidden rule to import something specific in a family from a generic component.